src/test/java/com/example/lesson6/zad3/EvenOrOddNumbersTest.java

```diff
@@ -0,0 +1,22 @@
+package com.example.lesson6.zad3;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class EvenOrOddNumbersTest {
+
+
+
+    @Test
+    void shouldEvenNumbers(){
+        boolean resultEvenNumbers = EvenOrOddNumbers.getEvenNumbers(20);
+        Assertions.assertEquals(true,resultEvenNumbers);
```

Comment:
zachowaj proszę konwencję 
`//given
//when
//then`
zamiast metody assertEquals możesz to uprościć używając metody Assertions.assertTrue